lib/widgets/nt_widgets/multi-topic/yagsl_swerve_drive.dart

```diff
@@ -65,11 +79,15 @@ class YAGSLSwerveDriveModel extends NTWidgetModel {
       ...super.toJson(),
       'show_robot_rotation': _showRobotRotation,
       'show_desired_states': _showDesiredStates,
+      'angle_offset': _angleOffset,
     };
   }
 
   @override
   List<Widget> getEditProperties(BuildContext context) {
+    String angleUnit =
+        tryCast(ntConnection.getLastAnnouncedValue(rotationUnitTopic)) ??
+            "radians";
```

Comment:
I don't think the angle unit should be dependent on what the topic publishes, since otherwise the angle offset will be changing when connected and disconnected

I think it should be kept as degrees and have it converted to radians when applying the offset

Reply:
Ok,I'll make those changes